template/codeland.rb

```diff
@@ -194,11 +199,21 @@ def user_not_authorized
     end
   end
 
+  empty_directory 'decorators' do
+    create_file '.keep'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.